lib/dev_toolbar/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module DevToolbar
-  VERSION = "1.2.0"
+  VERSION = "2.0.0"
```

Comment:
My thought was that the addition of the entire engine structure and movement of the ERD route from `/erd.png` to `/erd`, along with the required changed location of the file from `public/erd.png` to the root folder, and also the renaming of the link from "Data Model" to "ERD", warranted a major version update.

I.e., per [semvar](https://semver.org/), this seemed like a breaking change without backward compatibility.

That said, I find I often struggle to decide between X.Y.Z when making version bumps, so I'm open to arguments / suggestions here for why this would by a Y or Z and not an X bump for this change. Even just some good resources you could point me towards.